src/python/turicreate/data_structures/serialization.py

```diff
@@ -8,6 +8,7 @@
 # SFrames require a disk-backed file or directory to work with.  This directory 
 # has to be present to allow for serialization or deserialization. 
 __serialization_directory = None 
+from sys import version_info
```

Comment:
You're already importing this in the function which it is used. So you don't need this one. 

If you do want to keep this one, please hide.